pkg/storage/redis/redis.go

```diff
@@ -208,6 +210,50 @@ func (db *DB) ClearDeleted(_ context.Context, cutoff time.Time) {
 	}
 }
 
+// Sync returns a channel to the caller, when there is a change to the version set,
+// sending message to the channel to notify the caller.
+func (db *DB) Sync(ctx context.Context) chan struct{} {
```

Comment:
nit: maybe call this something else to avoid confusion?  Sub or Watch?

Reply:
I'm open to any better naming suggestion.

Reply:
@calebdoxsey @desimone Unless you have a better name, I will go with @travisgroth for `Watch`.